rmgpy/molecule/adjlist.py

```diff
@@ -695,9 +697,12 @@ def from_adjacency_list(adjlist, group=False, saturate_h=False):
         if group:
             atom = GroupAtom(atom_type, unpaired_electrons, partial_charges, label, lone_pairs, props)
         else:
-            atom = Atom(atom_type[0], unpaired_electrons[0], partial_charges[0], label, lone_pairs[0])
-            if isotope != -1:
-                atom.element = get_element(atom.number, isotope)
+            try:
+                atom = Atom(atom_type[0], unpaired_electrons[0], partial_charges[0], label, lone_pairs[0])
+                if isotope != -1:
+                    atom.element = get_element(atom.number, isotope)
+            except KeyError:
+                atom = CuttingLabel(name=atom_type[0], label=label)
```

Comment:
Minor: instead of try except, is there a way we can look at the atom information to tell in advance that it must be a cutting label, and then use if/else statements to handle this case? It is possible that other `KeyError`s could get thrown from this, which would make debugging hard.

Reply:
Sure, just modified it to if/else and detecting cutting label